fem/transfer.cpp

```diff
@@ -522,6 +522,439 @@ const Operator &L2ProjectionGridTransfer::BackwardOperator()
    return *B;
 }
 
+
+L2ProjectionH1GridTransfer::L2ProjectionH1::L2ProjectionH1(
+   const FiniteElementSpace& fes_ho_, const FiniteElementSpace& fes_lor_)
+   : Operator(fes_lor_.GetVSize(), fes_ho_.GetVSize()),
+     fes_ho(fes_ho_),
+     fes_lor(fes_lor_),
+     P(NULL),
+     p_rtol(1e-13),
+     p_atol(1e-13)
```

Comment:
I added a `SetTolerance()` method, but this might not be the best way to expose this to a user?  It would be nice to set this upon construction of `L2ProjectionGridTransfer::L2ProjectionH1Space`, but that seems like it would require changing the interface for `ForwardOperator()` and `BackwardOperator()`, which also isn't ideal.

Reply:
Resolved.  Added `SetRelTol()` and `SetAbsTol()` in latest commit.